sql_commands.txt

```diff
@@ -0,0 +1,8 @@
+1.) SELECT * FROM Players WHERE Team = 'NYK';
+2.) SELECT * FROM Players WHERE Team = 'IND' AND Age <= '26';
+3.) SELECT Sort_by(Points) FROM Players;
```

Comment:
Right idea sorting by column. In SQL the sorting command is `SORT BY` and it gets used after your `SELECT * FROM 'table_name'`:

`SELECT * FROM players ORDER BY points ASC;`

Also notice the `ASC`? That tells SQL that we want the rows to be presented in ascending order for this column (0, 1, 2, etc)